modules/manage/pages/cluster-maintenance/compaction-settings.adoc

```diff
@@ -28,10 +28,42 @@ Compaction policy may be applied to a cluster or to an individual topic. If both
 
 == Tombstone marker removal
 
-When all messages for a given key are deleted, the topic partition keeps one message, called a tombstone, that contains the key, and a null value. A tombstone indicates to clients that the messages have been deleted. How long Redpanda keeps these tombstones depends on the topic cleanup policy:
+When all messages for a given key are deleted, the topic partition keeps one message, called a tombstone, that contains the key, and a null value. A tombstone indicates to clients that the messages have been deleted. You can specify how long Redpanda keeps these tombstones using both a cluster configuration property `tombstone_retention_ms` and a topic configuration property `delete.retention.ms`. If both are set, the topic-level tombstone retention policy overrides the cluster-level policy.
```

Comment:
>When all messages for a given key are deleted, the topic partition keeps one message, called a tombstone, that contains the key, and a null value. 

See other comment about why this is not accurate to what a tombstone record is- this is almost a correct view, but is misleading. Tombstones are not generated by regularly scheduled garbage collection, they are produced by Kafka clients.

>A tombstone indicates to clients that the messages have been deleted.

No, it is a request by the client that previous records with the same key _should_ be deleted.